src/tracing.rs

```diff
@@ -78,13 +78,7 @@ impl Tracing {
         // and stash it in `INSTALL`
         INSTALL
             .get_or_init(|| {
-                let format = tracing_subscriber::fmt::format().without_time().pretty();
-                let subscriber = tracing_subscriber::fmt()
-                    .with_max_level(Level::TRACE)
-                    .with_writer(MakeJsLogWriter::new())
-                    .with_ansi(false)
-                    .event_format(format)
-                    .finish();
+                let subscriber = make_tracing_subscriber(None);
```

Comment:
https://github.com/matrix-org/matrix-rust-sdk/pull/2856 changed the type of `JsLoggingSubscriber`, and we may as well just use `make_tracing_subscriber` rather than re-inventing it here.